[service] : Added support for perp http://b0llix.net/perp/.

hackman opened the pull request:
Added support for perp supervision daemon. We try to detect if
the service is managed by perp and if so we are issuing the proper
commands to manage it.

Signed-off-by: Dimitar Ianakiev <dimitar.q@siteground.com>

##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->

<!---
If you are fixing an existing issue, please include "Fixes #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->

##### ANSIBLE VERSION
<!--- Paste verbatim output from “ansible --version” between quotes below -->
```

```


##### ADDITIONAL INFORMATION
<!---
Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful.
  -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```

```

bcoca commented:
we are moving systems out of the service module, see `systemd` module, push this as a separate module and make sure facts can detect this as a service manager.

dimitarq committed:
module: perp
 Added support for perp(http://b0llix.net/perp/) supervisor daemon.

Signed-off-by: Dimitar Ianakiev <dimitar.q@siteground.com>

dimitarq committed:
module: perp
 Added RETURN documentation.

Signed-off-by: Dimitar Ianakiev <dimitar.q@siteground.com>

dimitarq committed:
module: perp
 Added boilerplate for "__metaclass__" and "from __future__ import".

Signed-off-by: Dimitar Ianakiev <dimitar.q@siteground.com>

dimitarq committed:
module: perp
 Deprecated get_exception.

Signed-off-by: Dimitar Ianakiev <dimitar.q@siteground.com>

dimitarq committed:
module: perp
 Fixing CI error "E107 Imports should be directly below
DOCUMENTATION/EXAMPLES/RETURN/ANSIBLE_METADATA".

Signed-off-by: Dimitar Ianakiev <dimitar.q@siteground.com>

mitaka commented:
Guys I need help on the failing ci tests. I have checked multiple times and I do believe that my imports are placed directly below DOCUMENTATION/EXAMPLES/RETURN/ANSIBLE_METADATA section. Please advise how to proceed from here.
